docs/product/rollback/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Rollback
+sidebar_order: 10000
+description: "Learn about Rollback, a personalized wrap-up for each team member to look back at their year with Sentry."
+---
+
+Sentry Rollback is a personalized wrap-up for each team member to look back at their year with Sentry. They will find illustrated content about how they and their organization used Sentry in 2024.
```

Comment:
The first sentence here is repeated from the description above, could you find a way to reword this or share some different information here?